app/src/common/urls.js

```diff
@@ -97,6 +100,11 @@ export const URLS = {
     `${urlBase}project/${activeProject}/usernames?filter.cnt.users=`,
   projectIndex: (activeProject) => `${urlBase}project/${activeProject}/index`,
 
+  projectStatus: (activeProject, interval) =>
+    `${urlBase}project/list/${activeProject}/${getQueryParams({
```

Comment:
`${activeProject}${getQueryParams({` - slash is not needed for query parameters

Reply:
Fixed